MekHQ/src/mekhq/campaign/mission/AtBDynamicScenarioFactory.java

```diff
@@ -1301,66 +1318,6 @@ public static Entity createEntityWithCrew(String faction, int skill, Campaign ca
         return en;
     }
 
-    /**
-     * Generates a new Entity without using a RAT. Useful for "persistent" or fixed units.
-     *
-     * @param name            Full name (chassis + model) of the entity to generate.
-     * @param fName            Faction code to use for crew name generation
-     * @param skill            RandomSkillsGenerator.L_* constant for the average force skill level.
-     * @param campaign
-     * @return                A new Entity
-     */
-    @Deprecated //Deprecated 15-Feb-2020 as unused, needs the current name system to be implemented if activated
-    private static Entity getEntityByName(String name, String fName, int skill, Campaign campaign) {
```

Comment:
Let's leave this here. It is extremely useful for debugging, or if you want to generate some specific units.

Reply:
This will need to be modified then, as it doesn't work properly as it is currently because of personnel changes, and has not since February. If you want me to I can restore it and fix it?

Reply:
Refactored it down so that it just uses the createEntityWithCrew but first grabs the MechSummary. This keeps the current behaviour while being less code.